docs/src/fit.md

```diff
@@ -60,6 +60,7 @@ The `fit_mle` method has been implemented for the following distributions:
 - [`Multinomial`](@ref)
 - [`MvNormal`](@ref)
 - [`Dirichlet`](@ref)
+- [`ProductDistribution`](@ref)
```

Comment:
The type is an internal implementation detail and not exported, the official API is `product_distribution`.
```suggestion
```